upgrade_analysis/__manifest__.py

```diff
@@ -20,7 +20,11 @@
     ],
     "installable": True,
     "external_dependencies": {
-        "python": ["dataclasses", "odoorpc", "openupgradelib"],
+        "python": [
+            "dataclasses",
+            "git+https://github.com/OCA/odoorpc.git@master",
```

Comment:
Why forcing this here?

Reply:
Otherwise, it will be erased by new repo tools, when merging.

Reply:
Yeah, but I think this is the cause of the red CI.

Reply:
Put the url link in test-requirements.txt. but it would be better to publish odoorpc and open upgrade lib to pypi. It's easy to automate.

Reply:
To say a bit more now that I have a keyboard: we should never use URLs in python external dependencies because it would force all users of the module to deploy with the exact same URL. And people deploying from git will probably want to use their own branch or pin a specific commit. This is basically for the same reason that we must never pin a project version in external dependencies.

Reply:
thanks